.github/workflows/internal-tests.yml

```diff
@@ -18,16 +18,20 @@ jobs:
     # Skip if this is an external contribution. GitHub secrets will be empty, so the step would fail anyway.
     if: ${{ github.event_name != 'pull_request' || !github.event.pull_request.head.repo.fork }}
     runs-on: ubuntu-latest
+    env:
+      TARGET_OWNER: clockworklabs
+      TARGET_REPO: SpacetimeDBPrivate
```

Comment:
We could make this a github secret if we don't want to include it in text here, but it's not really sensitive.